README.md

```diff
@@ -138,6 +138,29 @@ When testing out COVID-Net with PACS integration, be sure to delete any PACS fil
 python3 upload_swift_notify_cube.py
 ```
 
+Currently, the app does not support updating/editing the DICOM images directly. But as an alternative, a connection can be made to the PACS files on Swift, using a database environment (such as DataGrip). From there, locate `pacsfiles_pacsfile` and delete the desired DICOMs (which later be updated).
+
+Next, with only the updated DICOM files (that are to update the deleted ones) in `/covidnet_integration/images`, follow the same steps of uploading as above by running the python command above.
+
+#### Creating/Editing DICOM images
+
+To edit the header tag info of pre-existing DICOM images (such as patient name, age, or sex), there is a script, `dicom.py`, provided in the `covidnet_integration` project that allows users to set those values.
+
+With this script, a singular pre-existing DICOM image can be copied to create a new image file, with different header tag info. This can be used for testing purposes and for creating new patients (if the patient ID field is different, the DICOM will be assigned to that patient, and the user can then search for them).
+
+```
+1. Place the desired DICOM image into base_images folder, located in covidnet_integration
+2. Change any of the values for the `data` array object can be changed to whatever value desired, in dicom.py
+3. Run: python3 dicom.py
+4. Find the newly created copy of the DICOM in /covidnet_integration/images
+```
+
+Note: 
+```
+1. base_images and dicom.py must be in the same directory for the script to work
+2. dicom.py does not create new scans, it simply changes the header tag info for testing/display purposes on the UI for each patient
+```
+
```

Comment:
It might be better to keep this documentation in `covidnet_integration`.

Please, double check with the stakeholders.

Reply:
Will talk with Janakitti more about where to put the script and documentation, as well as confirm with PMs